### DISCUSSION TITLE
Branding & Customization

### DISCUSSION THREAD

**Discussion by StackXCore** [AzuraCast/AzuraCast#7428] [Q&A]
Title: How to Customize Public Page and Browser Tab in AzuraCast

I'm looking to customize the public-facing page in AzuraCast. Specifically, I would like to:

1. Change the title displayed in the browser tab. Currently, it shows "Station Name | AzuraCast." How can I edit this?
2. Modify or remove the "Powered by AzuraCast" text on the public page. What are the steps to do this?
3. Where the path file location?

I'm using and installing Docker.

Any guidance on where to find and edit these settings would be greatly appreciated!

Thanks

**Comment by gAlleb** [comment]
https://domain.com/admin/branding

https://www.azuracast.com/docs/
